### PR TITLE
Input element - move padding/margin to component variables #178 - Feature/36081

### LINKED ISSUE #178: Input element - move padding/margin to component variables
We should move padding/margin to component variabless (lines: 12, 21, 46) and change name of variable `$input__padding` to `$input__field-padding `

### PR DESCRIPTION
**Pull request by Januszpl**
Fixes #178
https://dev.snowdog.pro/issues/36081

Add --inline mod for global label class. Won't change usage of label in other elements. However I already saw that we should check usage of label in other elements and fix implementation in those places for example radio input and perhaps other places where imho spacing between elements should be set per container (for example radio__handler in _radio.hbs). As usually we put label inline next to input and those elements are wrapped in container (div, li, etc).

### CODE REVIEW

File: components/02-elements/form/input/_input.scss [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
         display: flex;
         flex-flow: row nowrap;
         align-items: center;
-
-        .input__label {
-            margin-bottom: 0;
-            margin-right: $spacer--medium;
-        }
+    }
+    &__label {
```
Comment (Igloczek) [inline components/02-elements/form/input/_input.scss]: That's not a default styles of the label for input, it should be added only to the `inline` modificator.

File: components/02-elements/form/input/input.config.js [hunk 2]
```diff
@@ -18,7 +18,10 @@ module.exports = {
     {
       name: 'inline',
       context: {
-        class: 'input--inline'
+        class: 'input--inline',
+        label: {
+          class: 'label--inline'
```
Comment (Igloczek) [inline components/02-elements/form/input/input.config.js]: We can extend `label--inline` styles for `input--inline`, will be easier than adding an additional class.

File: components/01-globals/typography/label/_label.scss [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
-$label-margin-bottom: 8px !default;
-$label-margin-left  : 0 !default;
-$label-font-size    : $font-size-base !default;
-$label-color        : $color-secondary !default;
-$label-font-family  : $font-family-base !default;
-$label-font-weight  : $font-weight-normal !default;
-$label-line-height  : $font-line-height !default;
-$label-transition   : none !default;
+$label-margin        : 0 0 $spacer !default;
+$label-margin--inline: 0 !default;
```
Comment (Igloczek) [inline components/01-globals/typography/label/_label.scss]: Variables alligment.

File: components/02-elements/form/input/input.config.js (hunk 2 above)
Comment (Januszpl) [reply components/02-elements/form/input/input.config.js]: I tried to follow BEM and add modifier where it should be. YOu mean by adding
```
.input--inline {
    .label {
        ....
    }
}
```

File: components/01-globals/typography/label/_label.scss (hunk 3 above)
Comment (Januszpl) [reply components/01-globals/typography/label/_label.scss]: fixed

File: components/02-elements/form/input/_input.scss (hunk 1 above)
Comment (Januszpl) [reply components/02-elements/form/input/_input.scss]: moved

**Pull request merged**